assets/ui/taskSelectionScreen.ui

```diff
@@ -1,6 +1,6 @@
 {
     "type": "TaskSelectionScreenLayer",
-    "skin": "MasterOfOreon:plantScreen",
```

Comment:
Is this used anywhere else anymore or can it be removed?

Reply:
It is still used at the `plantSelectionScreen.ui`.

If we use the new file at the `plantSelectionScreen.ui`, then we can delete:
* `plantScreen.skin` from `MasterOfOreon/assets/skins`
* `plantScreenSkinBox.png` from `MasterOfOreon\assets\textures`
* `plantScreenSkinScroll.png` from `MasterOfOreon\assets\textures`

Shall I proceed with the above changes?

Reply:
Actually, I'll have to look more into if the `.png` assets are used somewhere else too.

Reply:
is the `plantSelectionScreen` still used?

Reply:
Yes, it is. 

On that note, I want to request a different structure for the lists, which will conclude in removing the `plantSelectionScreen` all togetehr. Could I make a new issue about it? or shall I descuss it over the Discord Channel first?

Reply:
Please create an issue for it.

Reply:
Got it!